server/services/index.ts

```diff
@@ -9,24 +9,28 @@ import SupportedPrisonsService from './prison/SupportedPrisonsService'
 import AppInsightsService from './AppInsightsService'
 import { buildAppInsightsClient } from '../utils/azureAppInsights'
 import { createRedisClient } from '../data/redisClient'
+import SmokeTestStore from '../data/cache/SmokeTestStore'
 
 export const services = () => {
   const { hmppsAuthClient, applicationInfo } = dataAccess()
   const appInsightsTelemetryClient: TelemetryClient = buildAppInsightsClient(applicationInfo)
   const userService = new UserService(hmppsAuthClient)
   const scanBarcodeService = new ScanBarcodeService(hmppsAuthClient)
+  const redisClient = createRedisClient()
```

Comment:
I think changes here are probably OK but setting up `redisClient` to be available via `dataAccess()` in line 15 might fit more in the new pattern used in the template